docs/src/pages/components/data-grid/rendering/rendering.md

```diff
@@ -214,6 +214,14 @@ This means that the number of rows will drive the height of the grid and consequ
 
 {{"demo": "pages/components/data-grid/rendering/AutoHeightGrid.js", "bg": "inline"}}
 
+### Infinite loading
```

Comment:
The demo is inside this navigation:

`# Data Grid - rendering > ## Layout > ### Infinite loading`

This doesn't sound right, especially the layout part.

Reply:
It seemd like a good place to put it. Another option IMO would be here https://material-ui.com/components/data-grid/rows/#feeding-data. After the `Feeding data` or after the `Updating rows`?

Reply:
From what I understand infinite loading isn't solving a layout problem 💢 

I would personally move it inside https://material-ui.com/components/data-grid/rows/#updating-rows as we feed data but also update them later on.